chapters/ch12.md

```diff
@@ -4,15 +4,15 @@
 
 > * Choose dev and test sets from a distribution that reflects what data you expect to get in the future and want to do well on. This may not be the same as your training data’s distribution.
 
-* Chọn tập phát triển và tập kiểm tra từ một phân phối phản ánh dữ liệu bạn dự tính nhận được trong tương lai và muốn hoạt động tốt trên nó. Phân phối này có thể không giống phân phối dữ liệu huấn luyện của bạn.
+* Chọn tập phát triển và kiểm tra từ một phân phối phản ánh được dữ liệu bạn dự tính nhận được trong tương lai và muốn hoạt động tốt với nó. Phân phối này có thể không giống phân phối trên dữ liệu huấn luyện của bạn.
```

Comment:
- Gợi ý dịch
> bạn dự tính ~~nhận được ~~ **_sẽ xử lý_** trong tương lai và muốn hoạt động tốt với ~~nó~~ chúng.

Reply:
```suggestion
* Chọn tập phát triển và kiểm tra từ một phân phối phản ánh được dữ liệu bạn dự tính sẽ cần xử lý trong tương lai và muốn hoạt động tốt với chúng. Phân phối này có thể không giống phân phối trên dữ liệu huấn luyện của bạn.
```
Mình thêm chữ "cần" để bảo toàn ý bị động của câu. Ý là mình không kiểm soát được thực tế khách hàng sẽ gửi cái gì lên cho mình nhận nên mình phải dự tính trước.